ie_serving/server/start.py

```diff
@@ -72,10 +72,10 @@ def serve(models, max_workers: int=1, port: int=9000):
         sys.exit(0)
 
 
-def start_web_rest_server(models, rest_port):
+def start_web_rest_server(models, rest_port, num_threads):
     d = PathInfoDispatcher({'/': create_rest_api(models)})
     server = WSGIServer(('0.0.0.0', rest_port), d,
-                        numthreads=1, request_queue_size=50)
+                        numthreads=num_threads, request_queue_size=50)

```

Comment:
Use constant `REQUESTS_QUEUE_SIZE` from `config.py`.

Reply:
No sure if `REQUESTS_QUEUE_SIZE `will be a good fit there. 
It defines max size of requests queue inside IR Engine. Both gRPC and Rest workers contribute to engine's requests queue. Also there might be multiple models and engines.
I suggest having another constant for queue sizes for servers with `REQUESTS_QUEUE_SIZE` only for IR Engines.